### PR TITLE
fix: add empty line after `flox show` hint for better visibility

### PR DESCRIPTION
**Pull request by ysndr**
fixes https://github.com/flox/flox/issues/891

### CODE REVIEW

File: cli/flox/src/commands/search.rs [hunk 1]
```diff
@@ -130,6 +130,7 @@ impl Search {
 
                         {suggestion}
                         {FLOX_SHOW_HINT}
+
```
Comment (mkenigs) [inline cli/flox/src/commands/search.rs:133]:
```suggestion

                        {FLOX_SHOW_HINT}
```

**Review by mkenigs — CHANGES_REQUESTED** [review]
Lol didn't expect to have to request changes on this one

**Comment by mkenigs** [comment]
Also you need a `            info!("\n{FLOX_SHOW_HINT}");` a few lines below

**Comment by ysndr** [comment]
oh, yeah i misunderstood the issue lol

**Review by mkenigs — APPROVED** [review]

**Pull request merged**